mailchimp-connector/backendScripts/actions/deleteNode.js

```diff
@@ -0,0 +1,51 @@
+loadScript("/mailchimp-connector/backendScripts/utils.js");
+
+var DeleteNode = function () {
+    let utils = new Utils();
+    let nodeUtil = utils.getNodeUtil();
+    let magnoliaContext = Java.type("info.magnolia.context.MgnlContext");
+    let propertyUtil = Java.type("info.magnolia.jcr.util.PropertyUtil");
+    let notification = Java.type("com.vaadin.ui.Notification");
+    let SUCCESS_DELETION_MESSAGE = 'Node was successfully deleted';
+    let FAILURE_DELETION_MESSAGE = 'Node Deletion from Mailchimp failed: ';
+
+    function deleteFromJCR(identifier) {
+        let session = magnoliaContext.getJCRSession(this.parameters.get("workspace"));
+        nodeUtil.getNodeByIdentifier(this.parameters.get("workspace"), identifier).remove();
+        session.save();
+        notification.show(SUCCESS_DELETION_MESSAGE, notification.Type.HUMANIZED_MESSAGE).setDelayMsec(5000);
```

Comment:
Use a constant for storing the delay value, it is easier to maintain since the same value is used across the whole file.